NEWS.md

```diff
@@ -1,8 +1,10 @@
+# eatTools 0.7.6
+
+* add argument checks using the checkmate package
 * bugfix in `halveString()` for splitting at the last instance of a regular expression
 * new function `rbind_fill_vector()` allows `rbind()` for non-dimensional vectors of unequal length, filling empty entries with NA

```

Comment:
NIT: vectors are unidimensional and not non-dimensional ;)

Reply:
Thanks, I changed it to "unidimensional"